src/internal/nft-api.ts

```diff
@@ -199,17 +203,45 @@ export async function* getNftsForContractIterator(
 export async function getOwnersForContract(
   config: AlchemyConfig,
   contractAddress: string,
+  options?:
+    | GetOwnersForContractWithTokenBalancesOptions
+    | GetOwnersForContractOptions,
   srcMethod = 'getOwnersForContract'
-): Promise<GetOwnersForContractResponse> {
-  const response = await requestHttpWithBackoff<
+): Promise<
+  GetOwnersForContractResponse | GetOwnersForContractWithTokenBalancesResponse
+> {
+  // TODO: remove `any` cast after backend is updated
+  const response: any = await requestHttpWithBackoff<
     GetOwnersForNftContractAlchemyParams,
     RawGetOwnersForContractResponse
   >(config, AlchemyApiType.NFT, 'getOwnersForCollection', srcMethod, {
+    ...options,
     contractAddress
   });
 
+  // HACK: cast `balance` field to string until NFT API makes the change
```

Comment:
Are we saving "TODO changes contingent on backend updates" anywhere in a list such that we can reference them (say, if you're out or end up on another project)?

Reply:
Ah this one isn't big enough to warrant an asana task, but should be wrapped up by @ayang-rkjs soon!